CHANGELOG.md

```diff
@@ -67,11 +67,11 @@ The changelog format is based on [Keep a Changelog](https://keepachangelog.com/e
 
 #### Fixed
 
-- Use AggregateComplete instead of AggregateBonded if minimalApproval is 1[#1344](https://github.com/symbol/symbol-desktop-wallet/issues/1344)
+- Use AggregateComplete instead of AggregateBonded if minimalApproval is 1[#1344](https://github.com/symbol/desktop-wallet/issues/1344)
```

Comment:
As best practice, use code tags around any function, variable or class names to improve the readability of your changelog. 

Use `AggregateComplete` instead of `AggregateBonded` if `minimalApproval = 1`[#1344](https://github.com/symbol/desktop-wallet/issues/1344)